assets/webpack/webpack.common.js

```diff
@@ -11,15 +11,7 @@ const VueLoaderPlugin = require('vue-loader/lib/plugin');
 const staticAssetOutputPath = path.resolve(__dirname, '../../priv/static');
 
 function getS3AssetURL(argv) {
-  let bucketName;
-  if (argv.mode === 'development') {
-    bucketName = 'plenario2-assets-dev';
-  } else if (argv.env && argv.env.staging) {
-    bucketName = 'plenario2-assets-staging';
-  } else {
-    bucketName = 'plenario2-assets';
-  }
-  return `https://s3.us-east-2.amazonaws.com/${bucketName}`;
+  return `https://s3.amazonaws.com/plenario2-assets`;
```

Comment:
The images couldn't be found because the `plenario2-assets` bucket didn't exist. When I created the bucket it had a new naming scheme that omitted the `us-east-2`. Instead of trying to convenience the developer by pointing to different buckets dynamically, I'd rather just overwrite the string manually and manage a single bucket.